modules/shared/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import {
+  Drawer,
+  DrawerBody,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  useDisclosure,
+  Text,
+  Flex,
+  Box,
+} from "@chakra-ui/react";
+import MenuLogo from "../Icons/MenuLogo";
+import Hamburguer from "../Icons/Hamburguer";
+import Link from "next/link";
+
+const Links = [
+  { title: "Colecciones", href: "/Collection" },
+  { title: "Sesión de estudio", href: "/SesiónEstudio" },
+  { title: "Tienda", href: "/Tienda" },
+  { title: "Perfil", href: "/Perfil" },
+];
+
+export default function Menu() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <>
+        <Flex justify="space-between" p={5}>
+          <Box>
+            <MenuLogo fill={"white"} width={18} height={23} borderColor="#1867FF" borderWidth={7} />
```

Comment:
deberiamos tomarlo del theme

```suggestion
            <MenuLogo fill={"white"} width={18} height={23} borderColor="primary.100" borderWidth={7} />
```